[2024-07-03T17:31:18.846+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T17:31:19.282+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_to_postgres.load_csv_to_postgres manual__2024-07-03T17:31:17.059271+00:00 [queued]>
[2024-07-03T17:31:19.310+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_to_postgres.load_csv_to_postgres manual__2024-07-03T17:31:17.059271+00:00 [queued]>
[2024-07-03T17:31:19.313+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-03T17:31:19.351+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_csv_to_postgres> on 2024-07-03 17:31:17.059271+00:00
[2024-07-03T17:31:19.367+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=354) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T17:31:19.371+0000] {standard_task_runner.py:63} INFO - Started process 363 to run task
[2024-07-03T17:31:19.369+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'csv_to_postgres', 'load_csv_to_postgres', 'manual__2024-07-03T17:31:17.059271+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/csv_to_postgres.py', '--cfg-path', '/tmp/tmpbadznz2t']
[2024-07-03T17:31:19.374+0000] {standard_task_runner.py:91} INFO - Job 61: Subtask load_csv_to_postgres
[2024-07-03T17:31:19.594+0000] {task_command.py:426} INFO - Running <TaskInstance: csv_to_postgres.load_csv_to_postgres manual__2024-07-03T17:31:17.059271+00:00 [running]> on host 9dffc1d7579f
[2024-07-03T17:31:20.691+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_to_postgres' AIRFLOW_CTX_TASK_ID='load_csv_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-07-03T17:31:17.059271+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-03T17:31:17.059271+00:00'
[2024-07-03T17:31:20.696+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T17:31:20.757+0000] {base.py:84} INFO - Using connection ID 'postgres_destino' for task execution.
[2024-07-03T17:31:20.802+0000] {base.py:84} INFO - Using connection ID 'postgres_destino' for task execution.
[2024-07-03T17:31:20.816+0000] {sql.py:598} INFO - Loaded 7 rows into categories so far
[2024-07-03T17:31:20.817+0000] {sql.py:611} INFO - Done loading. Loaded a total of 7 rows into categories
[2024-07-03T17:31:20.818+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-03T17:31:20.819+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T17:31:20.835+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=csv_to_postgres, task_id=load_csv_to_postgres, run_id=manual__2024-07-03T17:31:17.059271+00:00, execution_date=20240703T173117, start_date=20240703T173119, end_date=20240703T173120
[2024-07-03T17:31:20.859+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T17:31:20.887+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T17:31:20.890+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
